import/trello/importTrello.ts

```diff
@@ -135,6 +136,29 @@ function convert(input: Trello): Block[] {
 
             outCard.fields.contentOrder = [text.id]
         }
+
+        // Add Checklists
+        if (card.idChecklists && card.idChecklists.length) {
```

Comment:
This is minor, and may be stylistic, but in general lengths should be checked as > 0, i.e. ` if (card.idChecklists && card.idChecklists.length > 0) {`.

Reply:
That was another thing I was wondering about. I originally used `> 0` (habit from writing Go) but kept seeing people doing what I ended up doing :shrug: so I wasn't sure if there was some hidden syntactic thing or just a style thing. I definitely prefer the clarity of `> 0`.